frontend/src/containers/data-tool/content/index.tsx

```diff
@@ -1,8 +1,17 @@
+import { useSyncDataToolContentSettings } from '@/containers/data-tool/sync-settings';
+
+import Details from './details';
 import Map from './map';
-// import Table from './table';
 
 const DataToolContent: React.FC = () => {
-  return <Map />;
+  const [{ details }] = useSyncDataToolContentSettings();
```

Comment:
A bit picky here but I think renaming the variable to `showDetails` reflects better its nature (you can tell by the name is a boolean), up to you!

Reply:
I agree, `showDetails` is more readable. Done.